exampleCourse/questions/element/integerInput/question.html

```diff
@@ -40,39 +69,74 @@
-      <p> Within this variant, the different display options for <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-integer-input-element"><code>pl-integer-input</code></a>  on the page are displayed. Specifically, this question uses the block option (<code>display="block"</code>) to display the input field on its own line instead of being inline with the question. Moreover, the weight for answering this question is higher than its predecessors.</p>
-      <p> Solve for the value of $c$ given $c = {{params.a}} + {{params.b}}$ </p>
+      <p>
+        The size of the input box can be set with the attribute <code>size</code>, and the attribute
+        <code>show-help-text="false"</code> removes the button for the help popup. This box below
+        has <code>size=5</code> instead of the default <code>size=35</code>.
+      </p>
+      <p>Solve for the value of $c$ given $c = {{params.a}} + {{params.b}}$</p>
     </pl-question-panel>
-    <pl-integer-input answers-name="c_4" display="block" weight="4"></pl-integer-input>
+    <pl-integer-input answers-name="c_4" size="5" show-help-text="false"></pl-integer-input>
   </div>
 </div>
 
 <div class="card my-2">
   <div class="card-body">
-      <pl-question-panel>
-          <p> The size of the input box can be set with the attribute <code>size</code>, and the attribute <code>show-help-text="false"</code> removes the button for the help popup.  This box below has <code>size=5</code> instead of the default <code>size=35</code>. </p>
-      <p> Solve for the value of $c$ given $c = {{params.a}} + {{params.b}}$ </p>
+    <pl-question-panel>
+      <p>
+        This question uses the <code>blank-answer</code> to specify a value to be used if the user
+        leaves it blank.
+      </p>
+      <p>
+        Solve for the value of $c$ given $c = 0^n$ for $n = 1$ (leave the question blank if the
+        value is zero).
+      </p>
     </pl-question-panel>
-    <pl-integer-input answers-name="c_5" size="5" show-help-text="false"></pl-integer-input>
+    <pl-integer-input
+      answers-name="c_5"
+      allow-blank="true"
+      blank-value="0"
+      correct-answer="0"
+    ></pl-integer-input>
   </div>
 </div>
 
 <div class="card my-2">
   <div class="card-body">
     <pl-question-panel>
-      <p>This question uses the <code>blank-answer</code> to specify a value to be used if the user leaves it blank.</p>
-      <p>Solve for the value of $c$ given $c = 0^n$ for $n = 1$ (leave the question blank if the value is zero). </p>
+      <p>The integer can also be expected to be provided in a different base.</p>
+      <p>
+        Solve for the value of $c$ given $c = {{params.a16}}_{16} + {{params.b16}}_{16}$ (provide
+        your answer in base $16$).
+      </p>
     </pl-question-panel>
-    <pl-integer-input answers-name="c_6" allow-blank="true" blank-value="0" correct-answer="0"></pl-integer-input>
+    <pl-integer-input answers-name="c_6" base="16"></pl-integer-input>
```

Comment:
Changing the `answers-name` here will have unexpected results for any existing variants of this question. This is _probably_ ok since this is the example course, but I'd still rather err on the side of caution.

(aside: this is why semantic answer names are valuable: if you reorder the inputs, you don't have to break existing variants!)

Reply:
Ah yep. I think it'll be fine but I'll change the names back to be on the safe side.

Reply:
Updated in https://github.com/PrairieLearn/PrairieLearn/pull/6980/commits/3e748594029889b02fdf0a4bf2c8e392f5f2d348